server/src/main/java/org/elasticsearch/index/engine/LiveVersionMap.java

```diff
@@ -476,10 +476,18 @@ public long ramBytesUsed() {
     }
 
     /**
-     * Returns how much RAM would be freed up by refreshing. This is the RAM usage of the current version map. It doesn't include tombstones
-     * since they don't get cleared on refresh, nor the old version map that is being reclaimed.
+     * Returns how much RAM is used by refresh. This is the RAM usage of the current and old version maps.
      */
     long ramBytesUsedForRefresh() {
+        return maps.ramBytesUsed();
+    }
+
+    /**
+     * Returns how much RAM could be reclaimed from the version map. This is the RAM usage of the current version map, and could be
+     * reclaimed by refreshing. It doesn't include tombstones since they don't get cleared on refresh, nor the old version map that
+     * is being reclaimed.
+     */
+    long reclaimableRamBytes() {
         return maps.current.ramBytesUsed.get();
```

Comment:
On a second thought, I wonder whether we should stick to existing terminology and call this `refreshableRamBytes` instead of introducing a new term. We already have `getRefreshingBytes()`.

Reply:
`refreshableRamBytes` just by its own might be a bit vague, but what about `reclaimableRefreshRamBytes`?